examples/clkgen_in_hdl/test_clkgen.py

```diff
@@ -0,0 +1,74 @@
+# This file is public domain, it can be freely copied without restrictions.
+# SPDX-License-Identifier: CC0-1.0
+
+import cocotb
+from cocotb.clock import Clock
+from cocotb.triggers import Timer
+
+MAXCOUNT = 123456
+
+"""
+In this example, the testbench is split into
+an HDL part and a Python part.
+
+We call our HDL ``TOPLEVEL`` ``tb_hdl`` here
+to make clear it's the HDL part of the testbench
+(as implemented in ``tb.sv``/``tb.vhdl``).
+``TOPLEVEL`` is defined in the Makefile.
+
+The Python part of the testbench is this file.
+"""
+
+
+async def progress_reporter(tb_hdl):
+    """Report progress by printing intermediate counter values."""
+    while True:
+        # With the following Timer, we sleep exactly the right time
+        # to end up printing every 10000th counter value.
+        # We could also wake up on each counter value change and
+        # print the value if it was modulo 10000,
+        # but simulation performance would be much worse.
+        await Timer(10_000 * 10, "ns")
```

Comment:
```suggestion
        await Timer(10000 * 10, "ns")
```
Underscores were apparently added in 3.6: https://docs.python.org/3.6/whatsnew/3.6.html#whatsnew36-pep515

Reply:
Damn... they added a lot of QoL stuff in 3.6.

Reply:
Damn indeed... changed.